### ISSUE TITLE
explicit --> analytical

### ISSUE THREAD

**Issue by OrtnerMichael** [magpylib/magpylib#794]
While preparing my last talk I did some terminology research.

**explicit expression:**
An explicit expression is a specific type of analytical expression where the dependent variable is expressed explicitly in terms of the independent variable(s). In other words, it directly shows the dependent variable on one side of the equation.

which we do not have ofc.

**analytical expression**
An analytical expression is a mathematical expression that can be expressed using standard functions, operations, and constants in a finite number of steps. It usually involves functions such as polynomials, exponentials, logarithms, trigonometric functions, and others. Special functions fall under this category - depending on who you ask.

We chose "explicit expressions" but that was really bad naming on our behalf. We should use "anything that computes fast and accurately" - so GPT suggests to call them "efficient algorithms" which is not a really bad idea but also not exactly descriptive :).

**Comment by Alexboiboi** [comment]
Impressive!

**Comment by OrtnerMichael** [comment]
yes yes ... sorry for the spam ... this was set as a reminder in a high stress phase.

While preparing my last talk I did some terminology research.

**explicit expression:**
An explicit expression is a specific type of analytical expression where the dependent variable is expressed explicitly in terms of the independent variable(s). In other words, it directly shows the dependent variable on one side of the equation.

which we do not have ofc.

**analytical expression**
An analytical expression is a mathematical expression that can be expressed using standard functions, operations, and constants in a finite number of steps. It usually involves functions such as polynomials, exponentials, logarithms, trigonometric functions, and others. Special functions fall under this category - depending on who you ask.

We chose "explicit expressions" but that was really bad naming on our behalf. We should use "anything that computes fast and accurately" - so GPT suggests to call them "efficient algorithms" which is not a really bad idea but also not exactly descriptive :).
